src/main/java/org/broadinstitute/hellbender/utils/variant/VariantContextVariantAdapter.java

```diff
@@ -24,9 +35,48 @@
     public boolean isSnp() { return variantContext.isSNP(); }
     @Override
     public boolean isIndel() { return variantContext.isIndel(); }
+
+    @Override
+    public UUID getUUID() {
+        return uuid;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```

Comment:
What I'm hearing you say is "yes we should use VariantUtils.variantsAreEquals. Also, it has a bug: it should also check the UUID." Yes?